File: app/src/main/java/com/example/user/watched/pages/dashboard/di/DashboardContributeModule.java
package com.example.user.watched.pages.dashboard.di;


import com.example.user.watched.pages.dashboard.DashboardFragment;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public interface DashboardContributeModule {

    @ContributesAndroidInjector(modules = DashboardModule.class)
    DashboardFragment bindDashboardFragment();

}

File: app/src/main/java/com/example/user/watched/pages/dashboard/di/DashboardContributeModule.kt
package com.example.user.watched.pages.dashboard.di


import com.example.user.watched.pages.dashboard.DashboardFragment
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
interface DashboardContributeModule {

    @ContributesAndroidInjector(modules = [DashboardModule::class])
    fun bindDashboardFragment(): DashboardFragment

}